packages/tocco-ui/src/EditableValue/typeEditors/DurationEdit.js

```diff
@@ -16,7 +16,8 @@ class DurationEdit extends React.Component {
     this.hoursShadow = React.createRef()
     this.minutesShadow = React.createRef()
     this.state = {
-      ...this.millisecondsToDuration(props.value)
+      ...this.millisecondsToDuration(props.value),
+      focus: props.value >= 0
```

Comment:
focus is true when there is a value >= 0
That sounds confusing. Focus should be true when the field is focused. Maybe call the state `showUnits`?